site/docs/components/button/usage.mdx

```diff
@@ -40,8 +25,4 @@ import { Button } from "@salt-ds/core";
 
 ### Props
 
-| Name                  | Type      | Description                                                                                          | Default   |
-| --------------------- | --------- | ---------------------------------------------------------------------------------------------------- | --------- |
-| disabled              | `boolean` | If `true``, the button will be disabled.                                                             | -         |
-| focusableWhenDisabled | `boolean` | If `true``, the button will be fosusable when disabled.                                              | -         |
-| variant               | -         | The variant to use. “Primary” is the default value, and the other options are “secondary” and “cta.” | "primary" |
+:propsTable{src="../packages/core/src/button/Button.tsx"}
```

Comment:
I'm not sure it's worth adding this as it may change slightly if we don't use the Mosaic plugin (it will be probably something like `<PropsTable componentName="Button" />`)

Reply:
How about I keep it in as a commented out line for now? That way we have the full path in case we need it. Once we know how we're going to do props tables, we can amend this template accordingly.

Reply:
Sounds good!

Reply:
Actually, assuming that your props table PR #1911 to gets merged soon, I may as well update this to be:

```suggestion
<PropsTable componentName="ComponentName" />
```